explorer/explorerroutes.go

```diff
@@ -655,6 +667,8 @@ func (exp *explorerUI) TxPage(w http.ResponseWriter, r *http.Request) {
 		exp.Height() - tx.Confirmations,
 		exp.Version,
 		exp.NetName,
+		inout,
```

Comment:
Should error if `inout` is not either "in" or "out".

Reply:
My take was to let it default to no highlighting and not force an error.  Right now the only way it will highlight is if inout = "in" or "out" and the index specified is match-able on the page.  Given the way this is done sharing the same tx page as a non-highlighting link, we would need to error on anything but "in", "out" and "".  

Happy to do expicit url validity if you want however. What type of error do you want to throw?  something like:  ```No matching page, block, address or transaction that could be found
that transaction has not been recorded``` ?

Reply:
I'm sure the front end does the right thing, but the user can type in anything for the url.

I think a redirect back to the vanilla /tx/{txid} would be most user friendly, but really it should be a 404 of some kind